hexrdgui/llnl_import_tool_dialog.py

```diff
@@ -376,6 +374,19 @@ def detector_selected(self, selected):
             self.ui.instrument.setDisabled(selected)
             self.detector = self.ui.detectors.currentText()
             self.current_image_selection = self.detector
+            # If we have already loaded this detector's image, reload it
+            if self.detector in self.completed:
+                self.current_image_selection = self.detector
+                # Find the data and dark files for this detector
+                data_file = self.loaded_images[f'{self.detector} data']
+                dark_file = self.loaded_images[f'{self.detector} dark']
```

Comment:
Does this work for all instrument types (TARDIS, PXRDIP, and FIDDLE)?

Reply:
Yes, although PXRDIP uses the same image for all image plates